hack/istio/multicluster/deploy-kiali.sh

```diff
@@ -75,6 +81,7 @@ deploy_kiali() {
     --set auth.strategy="openid"                  \
     --set auth.openid.client_id="kube"            \
     --set auth.openid.issuer_uri="https://${kube_hostname}/realms/kube" \
+    --set auth.openid.insecure_skip_verify_tls="true" \
```

Comment:
This doesn't seem related to this change?

Reply:
I needed that otherwise I was having other keycloak issues.

Reply:
ok, I set that back explicitly to false (which is the default).